.github/workflows/nightly-integration-test.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+name: Nightly Build and Test with zarf@main
+on:
+  schedule:
+    - cron: '0 7 * * *' ## Every day at 0700 UTC
+  workflow_dispatch: # Allows manual triggering
+
+permissions:
+  contents: read
+
+jobs:
+  nightly-integration:
+    uses: ./.github/workflows/parallel-tests.yaml
```

Comment:
We'll likely need to use a different test suite or make a new one, as a few of them rely on packages we build during the initial tests, and somewhere in the chain we parse out the version of zarf from `go.mod` and then try to use that version in the `uds-bundle.yaml`. A typical zarf version upgrade PR updates this version automatically: 

- https://github.com/defenseunicorns/uds-cli/pull/1103/files#diff-a3cf35511c9c12330b39cc9d96833317ec7f1740866f98d14bc85f9c4e0dea9d

I can work on crafting a test that doesn't rely on that though